README.md

```diff
@@ -752,6 +752,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [coop](https://github.com/rakyll/coop) - Cheat sheet for some of the common concurrent flows in Go.
 * [delve](https://github.com/derekparker/delve) - Go debugger.
 * [fastlz](https://github.com/fromYukki/fastlz) - Wrap over [FastLz](http://fastlz.org/) (free, open-source, portable real-time compression library) for GoLang.
+* [filetype](https://github.com/h2non/filetype) - Small package to Infer the file type checking the magic numbers signature.
```

Comment:
Really minor, but is Infer meant to be capitalized in the middle of the sentence? :P